### ISSUE TITLE
http.core.ContainerServer and ransport.Server are missing

### ISSUE THREAD

**Issue by damianoporta** [ngallagher/simpleframework#15]
Hello,
I am following the tutorial http://www.simpleframework.org/doc/tutorial/tutorial.php
but i have a problem during the import of:

```
import org.simpleframework.http.core.ContainerServer;
import org.simpleframework.transport.Server;
```
The Classes are missing. Any workaround?

Thanks

**Comment by ngallagher** [comment]
They have been refactored and are not called SocketProcessor and ContainerSocketProcessor 


     On Friday, 21 August 2015, 10:16, Damiano Porta <notifications@github.com> wrote:
   

 Hello,
I am following the tutorial http://www.simpleframework.org/doc/tutorial/tutorial.php
but i have a problem during the import of:import org.simpleframework.http.core.ContainerServer;
import org.simpleframework.transport.Server;
The Classes are missing. Any workaround?Thanks—

**Comment by damianoporta** [comment]
@ngallagher perfect, so we have to update the tutorial... thank you so much!

**Comment by sudipm-mukherjee** [comment]
@ngallagher The tutorial still says about using
`org.simpleframework.http.core.ContainerServer;`
but that is not available. Can you please point me to some documents about how to use the refactored code..

**Comment by Art-B** [comment]
**tl;dr:** 

http://www.simpleframework.org/doc/tutorial/tutorial.php should be updated with: 

ContainerServer -> ContainerSocketProcessor
Server -> SocketProcessor

---

@sudipm-mukherjee judging by https://github.com/ngallagher/simpleframework/commit/d06ffe2e84fdeb1a5509702b69d5ebbe8a2f4c7c

which *ahem* "Make interfaces more sensible", it looks like:

ContainerServer -> ContainerSocketConnector
Server -> SocketConnector

and then in https://github.com/ngallagher/simpleframework/commit/0dd45a32b2b83afc58122627702739149f919e31 for "clarity"

ContainerSocketConnector -> ContainerSocketProcessor
SocketConnector  -> SocketProcessor
